flask_monitoringdashboard/database/stack_line.py

```diff
@@ -78,7 +78,9 @@ def get_grouped_profiled_requests(db_session, endpoint_id):
     """
     t = db_session.query(distinct(StackLine.request_id).label('id')). \
         filter(Request.endpoint_id == endpoint_id). \
-        join(Request.stack_lines).subquery('t')
+        join(Request.stack_lines).limit(100).subquery('t')
```

Comment:
this will always give the data on the first 100 requests instead of the most recent 100.